src/main/resources/application.yaml

```diff
@@ -1,9 +1,15 @@
+sftp:
+  username: ${SFTP_USERNAME:}
+  password: ${SFTP_PASSWORD:}
+  upload-url: ${SFTP_UPLOAD_URL:}
+  environment-path: 'prod'
 form-flow:
   disabled-flows:
     - flow: laDigitalAssister
       staticRedirectPage: /
   pdf:
     path: ~
+    map-file: parent-guardian-csv-map.yaml
```

Comment:
@bseeger @lkemperman-cfa wondering if this makes sense to move to something like 
``` 
  csv:
    path: ~
    map-file: parent-guardian-csv-map.yaml
```